src/styles/isomer-template.scss

```diff
@@ -12095,4 +12095,14 @@ a.navbar-item {
 .CodeMirror {
     height: calc(85vh - #{$header-height} - #{$footer-height});
     overflow-y: scroll;
+}
+
+.breadcrumb-container {
+    padding-left: 50px;
```

Comment:
Can we consider changing the values specified in `.breadcrumb-container` and `.page-content-body` to use `rem` instead of `px`? Here's a good blogpost on the difference between `px` vs `rem`: https://engageinteractive.co.uk/blog/em-vs-rem-vs-px

Reply:
Resolved in https://github.com/isomerpages/isomercms-frontend/pull/167/commits/8a670b3672735951ef98d942b1d91f72b9782778